doc/sphinx/Pacemaker_Explained/operations.rst

```diff
@@ -38,113 +38,154 @@ two operations for the same resource with the same name and interval.
-   |                |                                   |    single: role; action property                    |
-   |                |                                   |    single: action; property, role                   |
-   |                |                                   |                                                     |
-   |                |                                   | Run the operation only on node(s) that the cluster  |
-   |                |                                   | thinks should be in the specified role. This only   |
-   |                |                                   | makes sense for recurring ``monitor`` operations.   |
-   |                |                                   | Allowed (case-sensitive) values: ``Stopped``,       |
-   |                |                                   | ``Started``, and in the case of :ref:`promotable    |
-   |                |                                   | clone resources <s-resource-promotable>`,           |
-   |                |                                   | ``Unpromoted`` and ``Promoted``.                    |
-   +----------------+-----------------------------------+-----------------------------------------------------+
+   :widths: 2 2 3 4
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Default
+     - Description
+   * - .. _op_id:
+       
+       .. index::
+          pair: op; id
+          single: id; action property
+          single: action; property, id
+       
+       id
+     - :ref:`id <id>`
+     - 
+     - A unique identifier for the XML element *(required)*
+   * - .. _op_name:
+       
+       .. index::
+          pair: op; name
+          single: name; action property
+          single: action; property, name
+       
+       name
+     - :ref:`text <text>`
+     - 
+     - An action name supported by the resource agent *(required)*
+   * - .. _op_interval:
+       
+       .. index::
+          pair: op; interval
+          single: interval; action property
+          single: action; property, interval
+       
+       interval
+     - :ref:`duration <duration>`
+     - 0
+     - How frequently (in seconds) to perform the action. A value of 0 means
+       Pacemaker will decide when to execute the action (this is used to
+       configure other properties for the action). A positive value defines a
+       *recurring action*, which is typically used with ``name`` set to
+       :ref:`monitor <s-resource-monitoring>`. *(required)*
+   * - .. _op_role:
+       
+       .. index::
+          pair: op; role
+          single: role; action property
+          single: action; property, role
+       
+       role
+     - :ref:`enumeration <enumeration>`
+     - 
+     - If set, the operation configuration applies only on nodes where the
+       cluster thinks the resource should be in the specified role. This makes
```

Comment:
"is in the specified role" might be clearer; at first reading, "should be" might sound like "what should happen to the resource next." Thoughts?

Reply:
will reword